core/src/main/java/org/frankframework/http/SoapActionServiceListener.java

```diff
@@ -0,0 +1,128 @@
+/*
+   Copyright 2025 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package org.frankframework.http;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+import lombok.Getter;
+
+import org.frankframework.configuration.ConfigurationException;
+import org.frankframework.core.HasPhysicalDestination;
+import org.frankframework.core.ListenerException;
+import org.frankframework.core.PipeLineSession;
+import org.frankframework.receivers.Receiver;
+import org.frankframework.receivers.ServiceDispatcher;
+import org.frankframework.stream.Message;
+import org.frankframework.util.StringUtil;
+import org.frankframework.util.XmlBuilder;
+
+/**
+ * Listener that allows a {@link Receiver} to receive messages as a SOAP webservice.
+ * Messages are routed based on their SOAP Action header or namespace.
+ *
+ * <br/>For each request:<ul>
+ * <li>MIME headers are described in a 'mimeHeaders'-XML stored under session key 'mimeHeaders'</li>
+ * <li>Attachments present in the request are described by an 'attachments'-XML stored under session key 'attachments'</li>
+ * <li>SOAP protocol is stored under a session key 'soapProtocol'</li>
+ * <li>SOAP action is stored under a session key 'SOAPAction'</li>
+ * </ul>
+ * and for each response a multipart message is constructed if a 'multipart'-XML is provided in sessionKey specified by multipartXmlSessionKey.
+ *
+ * @author Niels Meijer
+ */
+public class SoapActionServiceListener extends PushingListenerAdapter implements HasPhysicalDestination {
+
+	private final @Getter String domain = "Http";
```

Comment:
```suggestion
	private static final @Getter String domain = "Http";
```

edit: maar wordt helemaal niet gebruikt?

Reply:
Tja, dit is iets dat in een interface zit, en wordt gebruikt voor de console, zie https://example.frankframework.org/#/connections.
Idealieter is dit een annotatie oid en niet een stom veld...